core/src/filter/resources/jenkins/model/jenkins-version.properties

```diff
@@ -1 +1,2 @@
-version=${build.version}
\ No newline at end of file
+version=${build.version}
+java.min.level=${java.level}
```

Comment:
@oleg-nenashev it is property file, but its name shows / tells a purpose. If you find a `mysql.properties` you expect to see _things_ related with MySQL, for example, a configuration, etc...

My suggestion would be to add a new entry in the `MANIFEST` and read from there the value.